packages/contracts-bridge/contracts/BridgeMessage.sol

```diff
@@ -20,7 +20,8 @@ library BridgeMessage {
         TokenId, // 1
         Message, // 2
         Transfer, // 3
-        DeprecatedFastTransfer // 4
+        DeprecatedFastTransfer, // 4
+        ConnextTransfer // 5
```

Comment:
is this better implemented as an extension of the transfer? or as a new message type?

Reply:
can you explain further what you mean?